app/javascript/components/Greetings/Greetings.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { getGreetings } from '../../redux/greetings/greetings';
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+const Greetings = () => {
+    const dispatch = useDispatch();
+    const data = useSelector((state) => state.greetings)
+    const { pending } = data
+    useEffect(() =>{
+      if(pending){
+        dispatch(getGreetings())
+      }
+    }, [])
+  return (
+    <div>
+      {data.greetings.map(greeting => {
+        return <p>{greeting.title}</p>
+      })}
+    </div>
+  )
+}
+
+export default Greetings
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/211636027-079cfb3d-39c9-4815-abac-eb4e0837c509.png)

- [ ] As you can see in the image above that there is not any message :fearful: from the api. Kindly fix this issue :blush: Your message should be printed on the page but i cannot see any :thinking:

Reply:
@HAMMAS-SALEEM kindly click the greetings link you will see the messages.

Reply:
Hey @Paul-js-hub Thanks for responding. I clicked on that link but it was not showing anything. If it is working on your side then can you attach a screenshot from your side?

Reply:
Hey, @HAMMAS-SALEEM thanks for your reply I have attached the screenshots below.

![greeting1](https://user-images.githubusercontent.com/64077816/211645636-7154b49e-30e7-4063-906d-c546bfdbd882.PNG)
![greeting2](https://user-images.githubusercontent.com/64077816/211645647-d178fef7-c25f-4285-bd1e-a5776f2cd832.PNG)

Reply:
### Hey @Paul-js-hub 
Hmm, Strange!!! If its working on your side then it's great. You don't need to bother on this change there might be a problem on my side. You leave this comment and work on your MIT.md file which is the below one 👇 😊

Reply:
Hey, @HAMMAS-SALEEM thanks